### DISCUSSION TITLE
How to open from a command line

### DISCUSSION THREAD

**Discussion by djloot** [tom-englert/RegionToShare#56] [General]
I would love to launch this from a macro button, actually.  
I tried launching from C:\Program Files\WindowsApps\15863TomEnglert.RegionToShare_1.6.5717.0_neutral__c1sj353grjxmm\RegionToShare
but it failed.

**Comment by tom-englert** [comment]
This is rather a question to Microsoft, how to manually launch store apps.

**Comment by tom-englert** [comment]
A simple solution: Download the source and  build your own .exe

**Comment by djloot** [comment]
Thank Tom for pointing me in the right direction.  I ran the application and then with task manager opened the location and researched based on your comments.  Long story short, here is the command:  
"explorer.exe shell:appsFolder\15863TomEnglert.RegionToShare_c1sj353grjxmm!App"
